README.md

```diff
@@ -286,8 +286,9 @@ defined as follows.
 **326** As long as an issue labelled "Trial" or "Judgement" exists, no player may start a turn, regardless of what
 other rules specify. Futhermore, any rule specifying that a pull request should be merged in a timely fashion shall
 be read as specifying that the pull request should be merged in a timely fashion, after all issues labelled "Trial"
-or "Judgement" are closed. Any player may ignore this rule, if and only if they have asked for permission by
-creating an issue and a simple majority of players is in favour of allowing this rule to be ignored.
+or "Judgement" are closed. Any player may start a vote in a separate issue to ignore a particular issue labelled 
+"Trial" or "Judgement" for purposes of this rule. 
+That issue will be ignored if a simple majority votes in favour of doing so.
```

Comment:
Implied, and this allows for stuff like abstaining (as in: "count-this-vote-as-if-i-don't-play")